doc/metadata.yaml

```diff
@@ -1,12 +1,16 @@
 site_title: LXD
 site_logo_url: https://linuxcontainers.org/static/img/containers.png
 navigation:
-    - title: Getting started
+    - title: LXD
       location: index.md
+    - title: Getting started
```

Comment:
Removed the link on the "Getting started" group, to be consistent with other groups. The link will still work (since it was index.md).